src/codegen/function.rs

```diff
@@ -115,17 +128,42 @@ pub fn body_chunk(env: &Env, analysis: &analysis::functions::Info) -> Chunk {
     }
 
     let outs_as_return = !analysis.outs.is_empty();
+    let ret_array_length = if let Some(pos) = analysis.ret.parameter.as_ref().and_then(|p| p.array_length) {
+        // The first parameter is &self in case of methods
+        let pos_offset = if analysis.kind == library::FunctionKind::Method { 1 } else { 0 };
+        analysis.parameters.iter().nth((pos + pos_offset) as usize)
```

Comment:
clippy warning: called `.iter().nth()` on a Vec. Calling `.get()` is both faster and more readable